### DISCUSSION TITLE
Auth email lengths

### DISCUSSION THREAD

**Discussion by glowcap** [supabase/supabase#10709] [Questions] [answered]
I'm building out local validation to save some calls to Supabase and I'd like to check for max email username and domain lengths (or maybe the full address length)
`<username.length>@<domain.length>.pizza`

Are there limits for these and if so, what are they?

**Comment by GaryAustin1** [comment, marked as answer]
Supabase does not impose any limits on length that I know of.
The email protocol does have a limit for each part, which you should be able to google and see.
